git-coco

```diff
@@ -11,6 +11,34 @@ fail() {
     exit 1
 }
 
+url_parser() {
+    local url
+    local profile repo
+    local codecommit_regex="^codecommit::[a-z][a-z]-[a-z]+-[1-3]:\/\/"
+
+    url="$1"
+    
+    # codecommit:// url
+    if [[ $url =~ $codecommit_regex ]]; then
+        profile=$(echo $url | awk -F '[/@]' '{print $3}')
+        repo=$(echo $url | awk -F '[/@]' '{print $4}')
+
+        # If no profile set repo to what was parsed to profile otherwise set parsed profile to global profile
+        if [ -z "$repo" ]; then
+            repo=$profile
+        else
+            $_profile=$profile
```

Comment:
The left-hand side shouldn't have a dollar sign (`$`) here and quoting is usually a good idea

```suggestion
            _profile="$profile"
```